src/plugins/walletConnectToDapps/components/header/DappHeaderMenuSummary.tsx

```diff
@@ -0,0 +1,110 @@
+import { CloseIcon } from '@chakra-ui/icons'
+import { MenuGroup } from '@chakra-ui/menu'
+import { Box, HStack, Link, MenuDivider, MenuItem, VStack } from '@chakra-ui/react'
+import { CHAIN_NAMESPACE } from '@shapeshiftoss/caip'
+import dayjs from 'dayjs'
+import { useWalletConnect } from 'plugins/walletConnectToDapps/WalletConnectBridgeContext'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { RawText, Text } from 'components/Text'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { useEvm } from 'hooks/useEvm/useEvm'
+import { selectAssets } from 'state/slices/assetsSlice/selectors'
+import { selectSelectedLocale } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { DappAvatar } from './DappAvatar'
+
+export const DappHeaderMenuSummary = () => {
+  const { supportedEvmChainIds } = useEvm()
+  const selectedLocale = useAppSelector(selectSelectedLocale)
+
+  const assets = useAppSelector(selectAssets)
+  const translate = useTranslate()
+
+  const walletConnect = useWalletConnect()
+  const connectedEvmChainId = walletConnect.bridge?.connector.chainId
+  const chainName = useMemo(() => {
+    const name = getChainAdapterManager()
+      .get(
+        supportedEvmChainIds.find(
+          chainId => chainId === `${CHAIN_NAMESPACE.Evm}:${connectedEvmChainId}`,
+        ) ?? '',
+      )
+      ?.getDisplayName()
+
+    return name ?? translate('plugins.walletConnectToDapps.header.menu.unsupportedNetwork')
+  }, [connectedEvmChainId, supportedEvmChainIds, translate])
+  const handleDisconnect = walletConnect.disconnect
+
+  // 0x evm address
+  const connectedAccountAddress = walletConnect?.bridge?.connector.accounts[0] ?? ''
+
+  // will generalize for all evm chains
+  const accountExplorerLink = useMemo(() => {
```

Comment:
we may want pull this out to a hook if this gets used in more places through the UI?